cap-ci/config.yaml

```diff
@@ -0,0 +1,30 @@
+jobs:
+  - deploy
+  - smoke-tests
+  - cf-acceptance-tests-brain
+  - cf-acceptance-tests
+  - sync-integration-tests
+  - upgrade
+  - stratos
+backends:
+  - caasp4
+  - aks
+  - gke
+  - eks
+options:
+  - sa
+  - ha
+  - all
+schedulers:
+  - diego
+  - eirini
```

Comment:
need ability to deploy one or the other as well

Reply:
DONE.